x-pack/plugins/security_solution/public/detections/containers/detection_engine/rules/utils.ts

```diff
@@ -6,36 +6,54 @@
  */
 
 import { INTERNAL_IMMUTABLE_KEY } from '../../../../../common/constants';
+import { escapeKuery } from '../../../../common/lib/keury';
 import { FilterOptions } from './types';
 
+const SEARCHABLE_RULE_PARAMS = [
+  'alert.attributes.name',
+  'alert.attributes.params.index',
+  'alert.attributes.params.threat.tactic.id',
+  'alert.attributes.params.threat.tactic.name',
+  'alert.attributes.params.threat.technique.id',
+  'alert.attributes.params.threat.technique.name',
+];
```

Comment:
Was `subtechnique` not added as a searchable param? You know if there was any particular reason?

Reply:
Good catch, thanks, @spong! I'll add `subtechnique` to the searchable params as well.